arborist/server_test.go

```diff
@@ -1270,6 +1270,8 @@ func TestServer(t *testing.T) {
 
 		roleName := "bazgo-create"
 		policyName := "bazgo-create-b"
+		roleNameA := "bazgoA-create"
+		policyNameA := "bazgoA-create-b"
```

Comment:
you could probably create a 2nd policy without creating a 2nd role, if you want to simplify it